src/multicolorledbar.cpp

```diff
@@ -98,24 +101,32 @@ void CMultiColorLEDBar::changeEvent ( QEvent* curEvent )
     if ( curEvent->type() == QEvent::EnabledChange )
     {
         // reset all LEDs
-        Reset ( this->isEnabled() );
+        Reset();
     }
 }
 
-void CMultiColorLEDBar::Reset ( const bool bEnabled )
+void CMultiColorLEDBar::Reset()
 {
-    // update state of all LEDs
-    for ( int iLEDIdx = 0; iLEDIdx < NUM_STEPS_LED_BAR; iLEDIdx++ )
+    if ( eLevelMeterType == MT_LED )
     {
-        // different reset behavoiur for enabled and disabled control
-        if ( bEnabled )
-        {
-            vecpLEDs[iLEDIdx]->setColor ( cLED::RL_GREY );
-        }
-        else
+        // update state of all LEDs
+        for ( int iLEDIdx = 0; iLEDIdx < NUM_STEPS_LED_BAR; iLEDIdx++ )
         {
-            vecpLEDs[iLEDIdx]->setColor ( cLED::RL_DISABLED );
+            // different reset behavior for enabled and disabled control
+            if ( this->isEnabled() )
+            {
+                vecpLEDs[iLEDIdx]->setColor ( cLED::RL_GREY );
+            }
+            else
+            {
+                vecpLEDs[iLEDIdx]->setColor ( cLED::RL_DISABLED );
```

Comment:
Is this case used for anything?

Reply:
This is not used by the Jamulus software. When I wrote that class, I wanted to have something more general. But these things could be removed/cleaned up.

Reply:
I'll take the opportunity of further development to clean it out then. If I remember.